MapboxCoreNavigation.podspec

```diff
@@ -40,9 +40,10 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 79.0"
-  s.dependency "MapboxDirections", "~> 2.0"
+  s.dependency "MapboxNavigationNative", "~> 80.0"
+  s.dependency "MapboxDirections-pre", ">= 2.1.0-rc.1", "< 3.0.0"
   s.dependency "MapboxMobileEvents", "~> 1.0"
+  s.dependency "Turf", "~> 2.1.0"
```

Comment:
Why do we need to add Turf dependency? I think it's no longer required, as MapboxDirections links to it anyway.

Reply:
Directions SDK depends on Turf `~> 2.0` but in Nav SDK we will need Turf `2.1`.

Reply:
This is for #3466.